### ISSUE TITLE
Collect mock design files under one folder in the repository

### ISSUE THREAD

**Issue by kaanunsel** [bounswe/bounswe2026group11#52]
## 📋 Description
As part of the #42 ,three teams have created several mock design images. These should be uploaded to the repository for future accessibility.

- Directory path should be: `docs/design/<mobile|web>/mockup`.
- Every file name must be explanatory. For example: `login_page.png`
- 1 member from every team should open exactly one PR.
- Commit message convention: `docs(design): add scenario ... mockup designs`

**⏱️ Expected Time**: 1d

**📅 Due Date:** 20-02-2026

## ✅ Acceptance Criteria
- [x] Scenario 1 mocks are uploaded
- [x] Scenario 2 mocks are uploaded
- [x] Scenario 3 mocks are uploaded

## 🔗 References
#42

**Comment by kutayeroglu** [comment]
I think you have added me by mistake, @kaanunsel. 
Removed myself from assignees.
